web_database_rollback/controllers/main.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Authors: Cojocaru Marcel (marcel.cojocaru@gmail.com)
+#    Copyright (c) 2019 Cojocaru Aurelian Marcel PFA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import openerp
+import openerp.http as http
+from openerp.http import request
+
+
+class DBRollbackController(http.Controller):
+    @http.route(
+        '/web_database_rollback/activate',
+        type='json', auth='none')
```

Comment:
IIUC, this would allow anybody (even an unauthenticated user) to activate/deactivate the test mode. Maybe it would be safer to restrict it to authenticated users with a specific group?